### ISSUE TITLE
ncview 2.1.7 

### ISSUE THREAD

**Issue by powellb** [Homebrew/homebrew-science#3813]
### Indicate troubleshooting steps taken

- [ X] Ran `brew update` and retried your prior step?
- [ X] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
- [X ] Checked that a similar issue hasn't already been filed?
- [X ] Looked up the software bug tracker if reporting a build error?
- [ X] If you're seeing permission errors tried running `sudo chown -R $(whoami) $(brew --prefix)`?

### Bug reports / build errors:

I am using XQuartz 2.7.9, and when trying to open any netcdf file with ncview 2.1.7,  it fails with:

Error: attempt to add non-widget child "ccontourform" to parent "FILENAME" which supports only widgets

where FILENAME is any of a dozen variously created and sourced netcdf files.

% ncview mari_std_f.nc
Ncview 2.1.7 David W. Pierce  29 March 2016
http://meteora.ucsd.edu:80/~pierce/ncview_home_page.html
Copyright (C) 1993 through 2015, David W. Pierce
Ncview comes with ABSOLUTELY NO WARRANTY; for details type `ncview -w'.
This is free software licensed under the Gnu General Public License version 3; type `ncview -c' for redistribution details.

Note: no Ncview app-defaults file found, using internal defaults
Error: attempt to add non-widget child "ccontourform" to parent "mari_std_f.nc" which supports only widgets

Brew is fully up-to-date, and no errors reported by the doctor:
% brew doctor
Your system is ready to brew.
% brew info ncview
homebrew/science/ncview: stable 2.1.7 (bottled)
Visual browser for netCDF format files
http://meteora.ucsd.edu/~pierce/ncview_home_page.html
/usr/local/Cellar/ncview/2.1.7 (7 files, 2.2M) *
  Poured from bottle on 2016-04-03 at 19:35:46
From: https://github.com/Homebrew/homebrew-science/blob/master/ncview.rb
==> Dependencies
Required: netcdf ✔, udunits ✔

**Comment by tdsmith** [comment]
I see:

```
tim@rocketman:homebrew-core (master)$ ncview ~/Downloads/tos_O1_2001-2002.nc 
Ncview 2.1.7 David W. Pierce  29 March 2016
http://meteora.ucsd.edu:80/~pierce/ncview_home_page.html
Copyright (C) 1993 through 2015, David W. Pierce
Ncview comes with ABSOLUTELY NO WARRANTY; for details type `ncview -w'.
This is free software licensed under the Gnu General Public License version 3; type `ncview -c' for redistribution details.

Note: could not open file /Users/tim/.ncviewrc for reading
Note: no Ncview app-defaults file found, using internal defaults
```

Try removing ~/.ncviewrc.

**Comment by powellb** [comment]
Removing .ncviewrc doesn't help. Which XQuartz are you using?

**Comment by tdsmith** [comment]
You're right! Succeeds with 2.7.8, fails with 2.7.9.

**Comment by tdsmith** [comment]
It looks like it just needs to be rebuilt; installing it from source with `brew install -s ncview` works fine for me.

**Comment by powellb** [comment]
Thanks, that worked for me also. Feel free to close. I appreciate you looking into it.

**Comment by tdsmith** [comment]
We should refresh the bottle. Thanks for the report!

**Comment by k-atb** [comment]
`brew install -s ncview `worked for me also

**Comment by coreice** [comment]
Hmmm, freshly installed ncview today, also got this error. Only solution is using XQuartz 2.7.8 (I have 2.7.9)?

**Comment by tdsmith** [comment]
Still haven't refreshed the bottle, sorry! I'll do that today. In the meantime `brew install -s ncview` will work.

**Comment by tdsmith** [comment]
This should be closed by #3824.
